common/membership/ringpop/factory.go

```diff
@@ -123,10 +123,7 @@ func (factory *factory) getMonitor() *monitor {
 			factory.Logger.Fatal("Failed to get current cluster ID", tag.Error(err))
 		}
 
-		appName := "temporal"
-		if currentClusterMetadata.UseClusterIdMembership {
-			appName = fmt.Sprintf("temporal-%s", currentClusterMetadata.GetClusterId())
-		}
+		appName := fmt.Sprintf("temporal-%s", currentClusterMetadata.GetClusterId())
```

Comment:
this could cause downtime for release < 1.19. It this acceptable?

Reply:
> Sequential Upgrades: Temporal Server should be upgraded sequentially. ... This sequence should be repeated until your desired version is reached.

I was going to ask -- is this aligned with the upgrade guide?